python/doc/usecases/use_case_flood_model.rst

```diff
@@ -97,6 +149,276 @@ We want to estimate the flood probability:
+        grid.setGraph(i, 0, graph)
+    grid.setTitle("Default scenario: dyke is low")
+    view = otv.View(
+        grid,
+        figure_kw={"figsize": (7.0, 9.0)},
+        legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
+    )
+    pl.subplots_adjust(wspace=0.4, hspace=0.5, right=0.7)
+
+The next figure presents the Sobol' indices of the three outputs in the
+scenario where the height of the dyke is high.
+
+.. plot::
+
+    import openturns as ot
+    import openturns.viewer as otv
+    import pylab as pl
+    from openturns.usecases import flood_model
+
+    fm = flood_model.FloodModel(distributionHdLow=False)
+    outputDimension = fm.model.getOutputDimension()
+
+    sampleSize = 1000
+    sie = ot.SobolIndicesExperiment(fm.distribution, sampleSize)
+    inputDesign = sie.generate()
+    input_names = fm.distribution.getDescription()
+    inputDesign.setDescription(input_names)
+    inputDesign.getSize()
+    outputDesign = fm.model(inputDesign)
+
+    # Make a plot
+    grid = ot.GridLayout(outputDimension, 1)
+    grid.setTitle(f"n = {sampleSize}")
+    for i in range(outputDimension):
+        marginalOutputSample = outputDesign.getMarginal(i)
+        sensitivityAnalysis = ot.SaltelliSensitivityAlgorithm(
+            inputDesign, marginalOutputSample, sampleSize
+        )
+        graph = sensitivityAnalysis.draw()
+        graph.setTitle(marginalOutputSample.getDescription()[0])
+        grid.setGraph(i, 0, graph)
+    grid.setTitle("Scenario: dyke is high")
+    view = otv.View(
+        grid,
+        figure_kw={"figsize": (7.0, 9.0)},
+        legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
+    )
+    pl.subplots_adjust(wspace=0.4, hspace=0.5, right=0.7)
+
+The next figure presents the Sobol' indices of the height model output
+with four inputs :math:`(Q, K_s, Z_v, Z_m)` only.
+This is a simpler model that leads to a simplified analysis.
+
+.. plot::
+
+    import openturns as ot
+    import openturns.viewer as otv
+    import pylab as pl
+    from openturns.usecases import flood_model
+
+    fm = flood_model.FloodModel()
+    heightInputDistribution, heightModel = fm.getHeightModel()
+    outputDimension = heightModel.getOutputDimension()
+
+    sampleSize = 1000
+    sie = ot.SobolIndicesExperiment(heightInputDistribution, sampleSize)
+    inputDesign = sie.generate()
+    input_names = heightInputDistribution.getDescription()
+    inputDesign.setDescription(input_names)
+    inputDesign.getSize()
+    outputDesign = heightModel(inputDesign)
+
+    # Make a plot
+    sensitivityAnalysis = ot.SaltelliSensitivityAlgorithm(
+        inputDesign, outputDesign, sampleSize
+    )
+    graph = sensitivityAnalysis.draw()
+    graph.setTitle("Height model with (Q, K_s, Z_v, Z_m) as inputs")
+    view = otv.View(
+        graph,
+        figure_kw={"figsize": (6.0, 4.0)},
+        legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
+    )
+    pl.subplots_adjust(right=0.7)
+
```

Comment:
In the previous figures, one can see that the Sobol indices are not well estimated and some of them are negative. Maybe, it could be nice to increase the number of samples ?

Reply:
Very true. Fixed at 65c4e7cec5965261c2940003021a047ff0ab9d8e